src/main/java/com/thevoxelbox/voxelsniper/bukkit/BukkitVoxelSniper.java

```diff
@@ -66,4 +67,11 @@ public BukkitPlayer getPlayer(UUID uuid) {
     public Environment getEnvironment() {
         return Environment.BUKKIT;
     }
+
+    @Override
+    public Version getVersion() {
+        //todo: Does this work?
+        String version = "V" + Bukkit.getBukkitVersion().split("-")[0].replace('.' , '_');
+        return Version.valueOf(version);
```

Comment:
I do not think this would work for any version apart from the 'base' version. For example, 1.16 would probably work, but 1.16.5 won't

Reply:
Probably fixed

Reply:
I think it would now only take the first 1 in 1.xx. I would probably leave this for now, but be sure to test this properly when it all builds